template/src/dummy.scss

```
@@ -36,14 +38,14 @@ code {
   justify-content: center;
   min-height: 100vh;
 
-  background-color: #282c34;
+  background-color: application.$bg-color;
 
   color: #fff;
-  font-size: calc(10px + 2vmin);
+  font-size: calc(#{application.rem(10px)} + 2vmin);
```

Can't this just be the following?

```suggestion
  font-size: calc(#{rem(10px)} + 2vmin);
```

From my understanding `@forward` makes it available to use as-is.

---

To be tried, but from what I've read: while the "forward" import has no scope, the "use" import still has a scope — so the "application.scss" file has its default `application` scope.

Forward seems simply to gather many file imports into 1 single `use` (like a "helper" file).

---

That's correct @malparty the `@use` has the scope and the default scope is the file name, so @olivierobert @malparty how about we `@use 'src/assets/stylesheets/application' as *;` so that we don't need to namespace it? 🤔

---

IMO if we want to use `use ... as *`, then we can keep the good old `import` imports :D 
Because one of the main benefits of `use` is that we have a scope (and so we can keep simple variable names without the fear of naming conflicts). Feel free to challenge of course ^^

---

It is hard to make such a decision now as we don't usually `@import application.scss` 🙃  The current use case is rather contrived as it is for test purposes. So maybe leaving the default as it is now would be fine ⚖️